src/util.js

```diff
@@ -51,3 +54,6 @@ export var notEmpty = function(o) {
 export var sanitise = function(str) {
     return typeof str === 'string' ? str.replace(/</g, '&lt;').replace(/>/g, '&gt;') : str;
 };
+export var mergeArray = function(arr) {
```

Comment:
How about calling this function `flattenArray`? (I think that's more common name for this)

Reply:
done !